.github/workflows/integration-tests.yml

```
@@ -0,0 +1,269 @@
+name: Integration Tests
+
+on:
+  push:
+    branches: [ main, chralt98-add-it-tests ]
```

This is a pretty heavy workload. I propose we save resources and execute this only when a release candidate or release tag is set. In any case, `chralt98-add-it-tests` should be removed (for testing `dispatch` is a useful target). I can see at https://github.com/zeitgeistpm/zeitgeist/actions/runs/7448080700 that the CI works.

```suggestion
on:
  push:
    tags:
      - '^v[0-9]+.[0-9]+.[0-9]+(-rc[0-9]+)?$'
```